website/build.tsx

```diff
@@ -114,8 +115,9 @@ function Navbar({url}: NavbarProps): Element {
 		<nav id="navbar" class="navbar">
 			<div class="navbar-group">
 				<div class="navbar-item">
-					<a class={url === "/" && "current"} href="/">
-						Crank.js
+					<a class={`navbar-title-link ${url === "/" && "current"}`} href="/">
+						<img class="navbar-logo" src="/static/logo.svg" />
```

Comment:
Maybe this needs an alt tag to indicate that it's a visual cue to the text?

```suggestion
						<img class="navbar-logo" src="/static/logo.svg"  alt="" />
```